pandas/tests/scalar/timestamp/test_timezones.py

```diff
@@ -75,19 +69,6 @@ def test_tz_localize_ambiguous_bool(self, unit):
         with pytest.raises(pytz.AmbiguousTimeError, match=msg):
             ts.tz_localize("US/Central")
 
-        with pytest.raises(pytz.AmbiguousTimeError, match=msg):
-            ts.tz_localize("dateutil/US/Central")
-
-        if ZoneInfo is not None:
-            try:
-                tz = ZoneInfo("US/Central")
-            except KeyError:
-                # no tzdata
-                pass
-            else:
-                with pytest.raises(pytz.AmbiguousTimeError, match=msg):
-                    ts.tz_localize(tz)
-
```

Comment:
why delete this?

Reply:
This also got deleted when I merged with the main branch